Cargo.toml

```diff
@@ -28,18 +28,23 @@ license = "Apache-2.0"
 name = "up-client-mqtt5-rust"
 readme = "README.md"
 repository = "https://github.com/eclipse-uprotocol/up-client-mqtt5-rust"
-rust-version = "1.72"
+rust-version = "1.76"
 version = "0.1.0"
 
 [dependencies]
+async-std = { version = "1.12.0", features = ["attributes"] }
```

Comment:
It seems like to really see benefits of consolidating around a certain Rust async framework, we'd need to all share a runtime, which I doubt we will do between crates.

But just FYI, up-transport-zenoh-rust uses tokio as does up-transport-vsomeip-rust. up-streamer-rust uses async-std at the moment, but I planned at some point to migrate to tokio as well.

Reply:
I will look at consolidating between async-std and tokio, I agree that tokio might be the preferred async framework so will look into transitioning everything to that. I do know that up-rust is using async-std

Reply:
okay async-std has been removed from the dependency list